statistics_collector_lib/CMakeLists.txt

```diff
@@ -0,0 +1,96 @@
+cmake_minimum_required(VERSION 3.5)
+project(statistics_collector_lib)
```

Comment:
Can we align with `libcollector` everywhere?

Reply:
I'm not following: this is an attempt to rename in order to be more specific and avoid the name collision as @zmichaels11 noted above. I'm able to build the library and system_metrics_collector locally in a workspace with the name changes and waiting to see what CI says.

Reply:
I see, I was confused by the PR name. I'd be good to make the PR name match the content in the future.

Reply:
lib*

Reply:
libstatistics_collector

Reply:
Aligned on `libstatistics_collector`. Thanks.